config/plugins/webhooks/gtn/script.js

```diff
@@ -121,6 +121,9 @@
                     if (safe) {
                         persistLocation();
                     }
+                    // Add the class to the entire GTN page
+                    var body = $("body").addClass("galaxy-proxy-active");
+
```

Comment:
looking at this now I'm not sure it's right, should it be like this? I think I thought we were /inside/ the gtn embed (since jquery was in use) but reviewing this now we're accessing `gtn-embed` which is only on the outside.

I'm testing on eu, but `.contents()` is returning nothing

Reply:
```suggestion
                    // Add the class to the entire GTN page
                    document.getElementById("gtn-embed").contentDocument.getElementsByTagName('body').classList.add("galaxy-proxy-active")

```